js/basic_js.js

```diff
@@ -0,0 +1,58 @@
+
+function getSalary(salary) {
+    salary==Number
+    if (n==15) console.log('thuế', 0.3*n*10**6, 'lương ròng', (15-0.3*n)*10**6 )  
+    else if (n==7) console.log('thuế', 0.2*n*10**6, 'lương ròng', (7-0.2*n)*10**6) 
+    else if(n>7 && n<15) console.log('thuế', 0.1*n*10**6, 'lương ròng', (n-0.1*n)*10**6)  
+}
+function getAge(age) {
+    age==Number
+    if (n < 16)
+        console.log('hoc sinh khong du dieu kien')
+    else console.log('hoc sinh du dieu kien')
+}
+function checkNumber(n) {
+    n==Number
+    if (n==100) console.log('số bạn vừa nhập là 100')
+    else if (n < 100) console.log('số bạn vừa nhập là ', n, 'và nhỏ hơn 100')
+    else console.log('số bạn vừa nhập là', n, 'và lớn hơn 100')
+}
+function findMax(a,b,c) {
+    a,b,c==Number
+    var max = a 
+    if (b > max) max =b
+    if (c > max) max =c
+    console.log(max)
+}
+function getRank(score) {
+    score==Number
+    if (score >= 9 && score <=10) console.log('rank A')
+    else if (score >= 7 && score <= 9) console.log('rank B')
```

Comment:
```suggestion
    else if (score >= 7) return 'B'
```
Case if ở bên trên đã thỏa mãn mọi số lớn hơn >= 9 rồi, bên dưới sao phải kiểm tra nó nhỏ hơn 9 nữa. hơn nữa dòng trên còn sai (phải là nhỏ hơn 9 chứ không phải nhỏ hơn hoặc bằng 9). Tương tự cho các phần bên dưới